backend/src/main/java/matches/organizer/service/MatchService.java

```diff
@@ -17,10 +23,21 @@ public MatchService(MatchRepository matchRepository) {
         this.matchRepository = matchRepository;
     }
     public List<Match> getMatches() {
-        var anyPlayer = new Player("Arthur Friedenreich");
-        var anyMatch = new Match();
-        anyMatch.getPlayers().add(anyPlayer);
-        matchRepository.add(anyMatch);
-        return matchRepository.getAll();
+        try {
+            Player anyPlayer = new Player("Arthur Friedenreich");
+            Match anyMatch = new MatchBuilder()
+                    .setName("Nuestro partido")
+                    .setUserId(UUID.randomUUID())
+                    .setDate(LocalDate.now().plusDays(1))
+                    .setHour(LocalTime.now())
+                    .setLocation("La Bombonera")
+                    .build();
+            anyMatch.addPlayer(anyPlayer);
+            matchRepository.add(anyMatch);
+            return matchRepository.getAll();
+        }
+        catch (RuntimeException e) {
```

Comment:
¿Porqué no lanzar la excepción para afuera si ocurre? En todo caso si tenemos miedo de devolver un 500 y queremos modelar los errores, se puede usar algo llamado [error handlers](https://www.baeldung.com/exception-handling-for-rest-with-spring)